pkg/config/config.go

```diff
@@ -231,6 +231,9 @@ type ContainersConfig struct {
 	// Umask is the umask inside the container.
 	Umask string `toml:"umask,omitempty"`
 
+	// Use the container's name as default hostname instead of ID.
+	UseContainerNameAsHostName bool `toml:"use_container_name_as_hostname,omitempty"`
```

Comment:
naming wise I would remove the `Use` it adds no value IMO. ContainerNameAsHostName seems just as clear.

Reply:
We don't have a rule about it. I think in most cases we do it as author once the changes are addressed but I don't mind closing them as reviewer as well.